app/src/main/java/net/grandcentrix/blutufintegration/domain/GetDeviceUseCase.kt

```diff
@@ -0,0 +1,11 @@
+package net.grandcentrix.blutufintegration.domain
+
+import net.grandcentrix.blutufintegration.data.model.DeviceUiState
+import net.grandcentrix.blutufintegration.data.repo.BluetoothRepository
+
+class GetDeviceUseCase(private val bluetoothRepository: BluetoothRepository) {
+
+    val selectedDeviceFlow = bluetoothRepository.selectedDeviceStateFlow
```

Comment:
A usecase should be a class which only provides a function to execute which might return a result. It should not have any encapsulated state in the class nor expose other properties.